easybuild/easyconfigs/k/KneadData/KneadData-0.6.1-foss-2017a-Python-2.7.13-Java-1.8.0_162.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'PythonPackage'
+
+name = 'KneadData'
+version = '0.6.1'
+versionsuffix = '-Python-%(pyver)s-Java-%(javaver)s'
+
+homepage = 'https://pypi.python.org/pypi/kneaddata'
+description = """KneadData is a tool designed to perform quality control on metagenomic
+ sequencing data, especially data from microbiome experiments."""
+
+toolchain = {'name': 'foss', 'version': '2017a'}
+
+source_urls = [PYPI_LOWER_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['46540a22bebc84919a71138c419c69f68e9f5e35404587a2e31a3a3e53c66077']
+
+# KneadData does not itself use Java, but it has two dependencies that do, so
+# depending directly seems like an easy way to make sure that the versions
+# used by those deps are the same.
+
+dependencies = [
+    ('Python', '2.7.13'),
+    ('Java', '1.8.0_162', '', True),
+    ('Trimmomatic', '0.36', '-Java-%(javaver)s', True),
+    ('Bowtie2', '2.3.2'),
+    ('TRF', '4.09', '-linux64', True),
+    ('FastQC', '0.11.5', '-Java-%(javaver)s', True),
+    ('SAMtools', '1.6'),
+]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
+
+sanity_check_commands = [
+    ('kneaddata', '--version'),
+]
```

Comment:
Please make this single line, and use a plain string rather than a tuple (the latter is a bit confusing):

```python
sanity_check_commands = ["kneaddata --version"]